libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx

```diff
@@ -1,31 +1,33 @@
 import cl from 'clsx';
 import classes from './Label.module.scss';
```

Comment:
Maybe add a blank line between imports from packages and imports from within the project? To better see which imports are what? The same applies to Button.tsx and Tag.tsx too.

```suggestion

import classes from './Label.module.scss';
```